src/app/dialogs/tvl-dialog/tvl-dialog.component.html

```diff
@@ -9,40 +9,23 @@
 
   <div *ngIf="vt.isNonScoreboard()" id="top-div-light" class="dialog-card" [class]="vt.getThemeColor() + '-card'">
 
-
     <div *ngIf="data.type === 'income'">
       <div class="title">
         PS income history
       </div>
       <div #chart class="tvl-chart"></div>
     </div>
 
-    <mat-tab-group (selectedTabChange)="initChart($event)" *ngIf="data.type !== 'income'">
-
-      <mat-tab label="TVL">
-
-        <div class="title">
-          {{data.type == 'All' ? 'All pools' : data.type + ' pool'}} history
-        </div>
-        <div #chart class="tvl-chart"></div>
-
-      </mat-tab>
-
-      <mat-tab *ngIf="data.type !== 'PS' && data.type !== 'All'" label="Share Price">
-
-        <div class="title">
-          {{data.type}} Share price history
-        </div>
-        <div #chart class="tvl-chart"></div>
-
-      </mat-tab>
-
-    </mat-tab-group>
-
+    <div *ngIf="data.type !== 'income'">
+      
```

Comment:
can you please remove this unnecessary space?

Reply:
Done :)